inst/EpiModel2.rmd

```diff
@@ -17,30 +17,37 @@ suppressMessages(suppressWarnings(library(EpiModel)))
 
 ```{r echo = TRUE, eval = FALSE}
+# EpiModel 2.0
 num1 <- num2 <- 50
 nw <- network_initialize(n = num1+num2)
 nw <- set_vertex_attribute(nw, "group", rep(1:2, c(num1, num2)))
 ```
 
-Above we have the updated syntax for creating bipartite, now called "two-group", networks within the EpiModel workflow. The network is initialized as usual using `network_initialize` (a new version of `network.initialize` detailed in section "network_initialize, get_ and set_vertex_attribute"), however, in order to capture the previous two-mode structure, we set heterogeneity as a vertex attribute instead of a network feature using `set_vertex_initialize` (a new version of `set.vertex.attribute`); in the case of the new EpiModel workflow, this is done using the attribute `group`. This change allows for more flexibility both in interpretation as well as network structure: in the `bipartite` networks we have developed using EpiModel 1.x, mixing between the two modes is not allowed. However, in a two-group network, as group assignment is now a vertex attribute we can model within group mixing. For example, in a population of males and females, this new workflow allows us to model more than just heterosexual relationships as below:
+Above we have the updated syntax for creating bipartite, now called "two-group", networks within the EpiModel workflow. The network is initialized as usual using `network_initialize` (a new version of `network.initialize` detailed in section "Network Initialization Functions"), however, in order to capture the previous bipartite structure, we set heterogeneity as a vertex attribute instead of a network feature using `set_vertex_initialize` (a new version of `set.vertex.attribute`); in the case of the new EpiModel workflow, this is done using the attribute `group`. This change allows for consistency in interpretation as well as flexibility of network structure: in the bipartite networks we have developed using EpiModel 1.x, mixing between the two modes is not allowed. However, in a two-group network, as group assignment is now a vertex attribute, we can model within group mixing. For example, in a population of males and females, this new workflow allows us to model more than just heterosexual relationships as below:
```

Comment:
```suggestion
Above we have the updated syntax for creating bipartite, now called "two-group", networks within the EpiModel workflow. The network is initialized using `network_initialize` (a new version of `network.initialize` detailed in the section "Network Initialization Functions"). However, in order to capture the previous bipartite structure, we set heterogeneity as a vertex attribute instead of a network feature using `set_vertex_initialize` (a new version of `set.vertex.attribute`); in the case of the new EpiModel workflow, this is done using the attribute `group`. 

This change allows for consistency in interpretation as well as flexibility of network structure: in the bipartite networks developed using EpiModel 1.x, mixing between the two modes is not allowed. However, now that group assignment is a vertex attribute in a two-group network, we can model within-group mixing. For example, in a population of males and females, this new workflow allows us to model more than just heterosexual relationships:
```